docs/reference/migration/migrate_5_0.asciidoc

```diff
@@ -808,6 +808,12 @@ changed to now route standard output to the journal and standard error
 to inherit this setting (these are the defaults for systemd). These
 settings can be modified by editing the elasticsearch.service file.
 
+==== Longer startup times
+
+In Elasticsearch 5.0.0 the `-XX:+AlwaysPreTouch` flag has been added to the JVM
+startup options. This will increase the startup time of Elasticsearch as well as
+increasing the initial resident memory usage of the Java process.
```

Comment:
It might be worth adding a comment what we gain for the increased startup time though?